src/main/java/com/google/codeu/servlets/StatsPageServlet.java

```diff
@@ -0,0 +1,41 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.codeu.data.Datastore;
+import com.google.gson.JsonObject;
+
+/**
+ * Handles fetching site statistics.
+ */
+@WebServlet("/stats")
+public class StatsPageServlet extends HttpServlet{
+
+  private Datastore datastore;
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
+
+  /**
+   * Responds with site statistics in JSON.
```

Comment:
Can you give an example of the response in the comment here? Something like: `ex: {"messageCount": 10}`. That way someone who wants to use this endpoint won't need to read the code to know what the JSON object is going to look like.

Reply:
added along with newline in gitignore!